package.json

```diff
@@ -62,7 +62,8 @@
     "svg2vectordrawable": "2.9.1",
     "svgo": "3.3.2",
     "ts-jest": "29.3.2",
-    "tsx": "4.19.4"
+    "tsx": "4.19.4",
+    "undici": "7.9.0"
```

Comment:
this library is used internally by Node for HTTP requests, so we need it as a `devDependency` to modify the timeout

Reply:
Why did we need to modify the timeout?

Reply:
Possibly because I got rid of `axios` in favor of the built-in `fetch` which probably has a different default timeout